src/utils/common-utils/common-utils.spec.js

```diff
@@ -17,5 +20,32 @@ describe('common-utils', () => {
       expect(result).not.toBe(input);
       expect(result).toEqual(input);
     });
+
+    it('summary error should have focus and scroll into view', () => {
```

Comment:
I feel like this is part of the `deepClone` describe block by accident 🤔

Reply:
Also could you rephrase the test description so that the `it` makes sense as part of the sentence?

>it('should do this cool thing')

Reply:
Fixed